xbmc/application/Application.cpp

```diff
@@ -307,8 +307,7 @@ void CApplication::HandlePortEvents()
       {
         if (CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_fullScreen)
         {
-          CServiceBroker::GetWinSystem()->ResizeWindow(newEvent.resize.w, newEvent.resize.h,
-                                                       newEvent.move.x, newEvent.move.y);
+          CServiceBroker::GetWinSystem()->ResizeWindow(newEvent.resize.w, newEvent.resize.h, 0, 0);
```

Comment:
Is this correct? This call isn't OSX specific, do all platforms expect 0,0 ?

Reply:
This event is only used in osx (I added it a while ago). If you see the code just above windows does something similar and there's a TODO to move to this event.

Reply:
All good then